functions/src/index.ts

```diff
@@ -88,12 +84,12 @@ export const resetGame = onDocumentUpdated(
           value: "in_progress",
         });
 
-        return event.data?.after.ref.set(
-          {
-            value: 0,
-          },
-          {merge: true}
-        );
+        const snapshot = await collectionRef.get();
```

Comment:
Watch out! This `collectionRef` is not the one that we need to remove the docs from.

```suggestion
        const snapshot = await solvedWordsCollection.get();
```